tests/files/env_config.yml

```diff
@@ -0,0 +1,9 @@
+environments:
+  development:
+    prefix: Dev-Prefix-
+
+  staging:
+    prefix: Stage-Prefix-
+
+  production:
+    prefix: Prod-Prefix-
```

Comment:
Why do we need a separate environment config file? Couldn't these live in the environments themselves?